cockroachdb_dedicated/assets/dashboards/cockroach_cloud_dedicated_overview.json

```diff
@@ -1,6 +1,6 @@
 {
-  "title": "CockroachDB Dedicated Overview",
-  "description": "## CockroachDB Dedicated Overview\n\nThis dashboard provides a high-level view of your CockroachDB Dedicated cluster, including:\nA high-level view of SQL performance & latency.\n- Information about resource consumption to help aid in capacity planning.\n- Ability to drill down to specific nodes (identified by a (node, region) tag pair) within your cluster.",
+  "title": "CockroachDB Cloud Dedicated ",
```

Comment:
Hi, there is an accidental space after the title here `CockroachDB Cloud Dedicated`. Can we remove that?

Reply:
Thanks for pointing it out! I have addressed it in latest commit.